.gitignore

```diff
@@ -4,4 +4,7 @@
 /node_modules
 npm-debug.log*
 
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+.yarn/
+.yarnrc.yml
+package-lock.json
```

Comment:
Isn't `package-lock.json` a local config file? Why does it need to be in version control?

Reply:
Also btw `yarn start` doesn't work for me, I've been using `npm start` so maybe that has something to do with it

Reply:
`package-lock` is used by NPM to generate fully deterministic/repeatable builds, especially across different systems, so it's important to include in repositories that use NPM. It locks the dependency tree so that you always get the same versions for both your declared and transient dependencies. Without the `package-lock` file, you and I could technically end up with slightly different builds due to how NPM resolves dependencies. 

Weird to hear that `yarn` isn't working correctly. I personally used the Yarn installer for Windows (https://classic.yarnpkg.com/en/docs/install/#windows-stable), and it was able to work after restarting my terminal.

Reply:
Yeah I used the yarn installer too, and restarted the terminal and project, with nothing. Regardless, I just pushed a commit un-ignoring the package-lock file